### DISCUSSION TITLE
[Bug] Pin to Start in the default Windows 11 search does not pin a tile to the Windows 10 start menu

### DISCUSSION THREAD

**Discussion by AWarthog** [valinet/ExplorerPatcher#987] [General]
Pin to Start in the default Windows 11 search does not pin a tile to the Windows 10 start menu. Pretty minor overall, though figured I'd mention it.

Thanks for all of the work you've put in to help make Windows 11 a bit more usable.

Edition: Windows 11 Pro
Version: 21H2
OS build: 22000.527
Version of Explorer Patcher: 22000.493.42.2

![capture20220303200212302](https://user-images.githubusercontent.com/100887619/156681907-e4866720-b16c-4b3e-b5ec-50856e5b7bd5.png)

**Comment by valinet** [comment]
It’s explained why here: https://www.elevenforum.com/t/new-version-of-explorerpatcher.1991/post-96773
